lib/blockscore/base.rb

```diff
@@ -15,11 +17,11 @@ def initialize(options = {}, &block)
     def attributes
       return @attributes if @loaded
       force!
-      @attributes
+      attributes
     end
 
     def id
-      @attributes.fetch(:id, nil)
+      attributes.fetch(:id, nil)
```

Comment:
Given that you just return `nil` in the event the id is missing, I would use `Hash#[]` here instead.